project/cmake/scripts/rbpi/ArchSetup.cmake

```diff
@@ -18,6 +18,7 @@ else()
     set(ARCH arm-linux-gnueabihf)
```

Comment:
NEON is not set to true, so the option will default to off. do you want me to add an additional check in the common ArchSetup.cmake file? Not sure if that's necessary, cause if someone turns that open on, he likely knows what he's doing.